descwl_coadd/coadd.py

```diff
@@ -1266,7 +728,8 @@ def _get_coadd_psf_obs(self):
         get the psf observation
         """
 
-        coadd_cen = self.coadd_wcs.getPixelOrigin()
+        coadd_wcs = self.coadd_exp.getWcs()
+        coadd_cen = coadd_wcs.getPixelOrigin()
```

Comment:
This is the wrong center. We want the center of the coadd slice image.

Reply:
true, and I will fix

but we are storing a fixed kernel so it doesn't matter

it matters more for the jacobian a few lines up